core/src/skybox.cpp

```diff
@@ -0,0 +1,166 @@
+#include "knoting/skybox.h"
+
+namespace knot {
+namespace components {
+
+struct PosColorTexCoord0Vertex {
+    float m_x;
+    float m_y;
+    float m_z;
```

Comment:
Can this be a `vec3`?

Reply:
vec3 pos vertex class is obsolete and is handled by instanceMesh